### PR TITLE
Following our discussion, improving OCamlPro's visibility a little bit.

### PR DESCRIPTION
**Pull request by AltGr**
I've tried to be fair and not too invasive. Please don't shout if I failed :)

### CODE REVIEW

File: site/community/index.md [hunk 1]
```diff
@@ -51,13 +51,17 @@
...
-            <p>If you would like to give support to OCaml, you can join the Consortium or support the work of OCaml Labs. <a href="/community/support.html">Find out more</a>.</p>
+            <p>Besides the above <a href="/community/mailing_lists.html">mailing lists</a>, you
+            can ask questions on the OCaml IRC channels
+            (<a href="irc://irc.freenode.net/ocaml">en</a>,
+             <a href="irc://irc.freenode.net/ocaml-fr">fr</a>).</p>
+
+            <p>You can get <a href="support.html">Professional support</a> from
+            one of <a href="support.html"><img src="/img/ocamlpro.png"
+            alt="OCamlPro" style="height: 2ex; vertical-align: text-bottom" />,
+            Gerd Stolpman...</a>.</p>
```
Comment (avsm) [inline site/community/index.md:62]: This doesn't include (e.g. @dbuenzli) -- there used to be a comprehensive list of people offering consultancy services on one of the OCaml wikis, but I can't find it now...

File: site/docs/install.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ among other things, used by Debian to manage their packages. It works
 well on Unix, Linux, and Mac OS X systems. Windows support is comming
 soon.
 
+OPAM is edited by
```
Comment (avsm) [inline site/docs/install.md:27]: Edited isn't quite the right wording here.  Perhaps "Commercial support for OPAM is available from..." instead?

File: site/community/index.md (hunk 1 above)
Comment (amirmc) [inline site/community/index.md:62]: I don't think we should name anyone specific on this page.  That should happen on the support page.

File: site/docs/install.md (hunk 2 above)
Comment (amirmc) [inline site/docs/install.md:27]: I'd merge this sentence with the previous paragraph and reword it to something like "OPAM was created and is maintained by OCamlPro (commercial support is available)"  -- with a link to the support page.

File: site/community/index.md [hunk 3]
```diff
@@ -51,13 +51,17 @@
...
-			<a href="irc://irc.freenode.net/ocaml-fr">fr</a>).
-			When you need to go beyond this and get professional support, you have a number of
-			<a href="support.html">options available</a>.</p>
-            <p>If you would like to give support to OCaml, you can join the Consortium or support the work of OCaml Labs. <a href="/community/support.html">Find out more</a>.</p>
+            <p>Besides the above <a href="/community/mailing_lists.html">mailing lists</a>, you
+            can ask questions on the OCaml IRC channels
+            (<a href="irc://irc.freenode.net/ocaml">en</a>,
+             <a href="irc://irc.freenode.net/ocaml-fr">fr</a>).</p>
+
+            <p>You can get <a href="support.html">Professional support</a> from
```
Comment (amirmc) [inline site/community/index.md:59]: I have some suggestions for these paragraphs but I'll mention them in the PR thread later

File: site/community/index.md (hunk 1 above)
Comment (Chris00) [inline site/community/index.md:62]: > This doesn't include (e.g. @dbuenzli) -- there used to be a comprehensive list of people offering consultancy services on one of the OCaml wikis, but I can't find it now...

I thought so too.  Maybe it is time to ask the list again (so it is up-tp-date)?

**Comment by amirmc** [comment]
Thanks for the PR!  We've now made the new site live (announcement is still pending) so this PR should really be to the `master` branch.  

Overall, I think everything looks ok but there are some minor language improvements I'd like to make to the community page.

File: site/community/index.md (hunk 1 above)
Comment (amirmc) [inline site/community/index.md:62]: @Chris00 Good idea.  I'm writing an announcement email for the list later tonight.  I can ask people to look over the content and submit PRs for content updates, including people offering services.

**Comment by agarwal** [comment]
As a stylistic matter, I'm not sure I like the use of a logo mixed in with plain text. It breaks the consistent look and feel of the site.

File: site/community/index.md (hunk 1 above)
Comment (dbuenzli) [inline site/community/index.md:62]: In any case there's this page http://mirror.ocamlcore.org/wiki.cocan.org/people.html but it's strange since I used to have a line in the Europe section, maybe not the latest version was mirrored.

File: site/docs/install.md (hunk 2 above)
Comment (avsm) [inline site/docs/install.md:27]: Well, OCaml Labs does also spend a considerable amount of its time working on OPAM now, so I'd prefer a simple statement about the commercial support (which is definitely an OCamlPro exclusive).

**Comment by agarwal** [comment]
It says I closed this pull request, but I didn't mean to. I must have hit the wrong button. I don't see an option to open it again, like you can with Issues.

**Comment by agarwal** [comment]
Maybe it got closed by me due to me deleting the `redesign` branch.

**Comment by amirmc** [comment]
> Maybe it got closed by me due to me deleting the `redesign` branch.

That would make sense.  Sorry about that @AltGr! A new PR to `master` would be needed so we'd have had to close this anyway.  If it's ok, I'll email you tomorrow with edits for a new PR.  Hope that's ok.

_edit: I didn't anticipate that deleting branches automatically closes PRs_

**Comment by agarwal** [comment]
Note I opened #308 to track this Pull Request.

File: site/community/index.md (hunk 1 above)
Comment (AltGr) [inline site/community/index.md:62]: Sure (sorry @dbuenzli !) ; I just reproduced the list on the 'support' page

**Comment by AltGr** [comment]
Sure, np !

File: site/docs/install.md (hunk 2 above)
Comment (AltGr) [inline site/docs/install.md:27]: @avsm true enough. How about both are quoted ? Something like

> OPAM is written by OCamlPro and OCamllabs, the former also offering <a>commercial support</a>.

Comment (agarwal) [inline site/docs/install.md:27]: @avsm Have you decided how to write "OCaml Labs"? I see it written sometimes as "OCamlLabs" (without a space). Pick and we'll use it consistently.

Comment (amirmc) [inline site/docs/install.md:27]: It's "OCaml Labs" (that's how we refer to it on our website (http://www.cl.cam.ac.uk/projects/ocamllabs/).  We end up using ocamllabs in places where we need one word (tags etc).

Comment (avsm) [inline site/docs/install.md:27]: With a space.
